content/en/blog/_posts/2021-06-18-Announcing-Kubernetes-Upstream-Community-Group-Annual-Reports.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: blog
+title: "Announcing Kubernetes Upstream Community Group Annual Reports"
+date: 2021-06-18T10:00:00-08:00
+slug: Announcing-Kubernetes-Upstream-Community-Group-Annual-Reports
+---
+
+**Authors:** Divya Mohan
+
+Given the growth and scale of the Kubernetes project, the existing reporting mechanisms were proving to be inadequate & challenging. The sheer number of community groups (37 at the time of writing this), their varied sources & types of data that needed to be incorporated while preparing such updates, & the delivery of those (how many meetings are too many meetings?) clearly were a call to action for exploring better options. There are over 300 GitHub repositories alone and we count more than just commits to these repositories as a contribution. Therefore inspired by the Apache Software Foundation’s open guide to PMC Reporting and the CNCF project Annual Reporting, the Kubernetes project is proud to announce the **Kubernetes Community Group Annual Reports for Special Interest Groups (SIGs) and Working Groups (WGs)**. In its flagship edition, the [2020 Summary report]() focuses on bettering the Kubernetes ecosystem by assessing and promoting the healthiness of the groups within the upstream community. 
+
+As of today the existing mechanisms, typically, look at (devstats)[https://k8s.devstats.cncf.io/], GitHub data, issues, to measure the healthiness of a given UG/WG/SIG/Committee. As a project spanning several diverse communities, it was essential to have something that captured the human side of things. With 50,000+ contributors, it’s easy to assume that the project has enough help and this report surfaces more information than /help-wanted and /good-first-issue for end users. This is how we sustain the project. Paraphrasing one of the Steering Committee members, [Paris Pittman](https://github.com/parispittman), “There was a requirement for tighter feedback loops - ones that involved more than just Github data and issues. Given that Kubernetes, as a project, has grown in scale & number of contributors over the years, we have outgrown the existing reporting mechanisms."
```

Comment:
Maybe:&hellip;
```diff
-As of today the existing mechanisms…
+Previously, the mechanisms for the Kubernetes project overall to report on groups and their activities…
```
?

Reply:
I would write “and” not “&” within speech marks. I can't explain why but it's more idiomatic.

Reply:
Perhaps replace “devstats” with “CNCF devstats” as readers may never have heard of it.

Reply:
@parispittman would you be happy for us to adopt this draft quote as reported speech (ie, cut out the “Paraphrasing”)?

Reply:
+1 to @sftim's comments and suggestions above including `previously..` and CNCF devstats.